### ISSUE TITLE
Supermod's API_KEY access the broadcaster's chat

### ISSUE THREAD

**Issue by werrew** [teamrevlo/RevloAPI#7]
Hi~ Hello~

I'am supermod of the Broadcaster.

I want to use my API_KEY to access Broadcaster's chat.  (not use Broadcaster's API_KEY)

Is possible to add the parameter (EX:targetChat) in api ?

Thank you very much.

**Comment by scottnguyen** [comment]
Yes, we want to have more advanced role system in place sooner rather than later, but we're not sure where that will fit into our timeline just yet. For now, you should just use the broadcaster's API_KEY.

**Comment by Jinnboy** [comment]
Good!
Thank you very much. ^_^
